test/integration/integration_suite_test.go

```diff
@@ -79,7 +80,8 @@ func init() {
 	flag.StringVar(&manifest, "manifest-path", "./config/nephe.yml", "The relative path to manifest.")
 	flag.BoolVar(&preserveSetupOnFail, "preserve-setup-on-fail", false, "Preserve the setup if a test failed.")
 	flag.StringVar(&supportBundleDir, "support-bundle-dir", "", "Support bundles are saved in this dir when specified.")
-	flag.BoolVar(&withAgent, "with-agent", false, "Using antrea-agent on VM.")
+	flag.BoolVar(&withAgent, "with-agent", false, "Using antrea-agent on Linux VM.")
+	flag.BoolVar(&withWindows, "with-windows", false, "Using antrea-agent on Windows VM")
```

Comment:
add some validation that if with-windows is set, with-agent should be set too

Reply:
Done